code/modules/customitems/item_spawning.dm

```diff
@@ -7,17 +7,19 @@
 GLOBAL_LIST_FILE_LOAD(custom_items, "config/custom_items.txt")
 
 /proc/EquipCustomItems(mob/living/carbon/human/M)
+	var/client/donor = M.client
+	remove_verb(donor, /client/proc/claim_donor)
```

Comment:
no guarantee that this is being called from the verb, remove the verb from them inside that verb